spec/solver_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './solver'
+
+RSpec.describe 'Solver' do
+  it 'generates the factorial of a number' do
+    solver = Solver.new
+    factor = solver.factoria(5)
+    expect(factor).to eq 120
+  end
+  it 'generates the reversed string' do
+    solver = Solver.new
+    factor = solver.reverser_string('mane')
+    expect(factor).to eql 'enam'
+  end
+  it 'generates fizz or buzz' do
+    solver = Solver.new
+    factor = solver.fizzbuzz(5)
+    expect(factor).to eql 'Buzz'
+  end
```

Comment:
- [ ] We are missing to test our fizzbuzz when we passed the following numbers:
-  `a number which is a multiple of 3 itself`,
- `a non divisible with 3 and 5` and
- `a number which is a multiple of both 5 and 3`

Reply:
This issue is well resolved! ✔️